packages/runtime/src/useCases/consumption/attributes/DeleteOwnSharedAttributeAndNotifyPeer.ts

```diff
@@ -39,6 +40,12 @@ export class DeleteOwnSharedAttributeAndNotifyPeerUseCase extends UseCase<Delete
             return Result.fail(RuntimeErrors.attributes.isNotOwnSharedAttribute(ownSharedAttributeId));
         }
 
+        const canSendMessageResult = await this.relationshipsController.canSendMessage(ownSharedAttribute.shareInfo.peer);
+
+        if (canSendMessageResult.isError) {
+            return Result.fail(RuntimeErrors.attributes.cannotDeleteSharedAttributeWhileRelationshipIsPending());
```

Comment:
the error message is misleading, nothing idicates that canSendMessage failed because of a pending relationship. I think we can generalize that by saying `can't delete because we can't send a message`